README.md

```diff
@@ -3,14 +3,45 @@
 This repository contains assets that are binary files or too large to be checked in directly. Packages are uploaded to the default transport channel's feed.
 
 ## Workflow
-1. Submit a PR with the new assets.
-2. After the PR is merged, an internal build publishes assets to the Build Assets Registry and an auto PR will be opened in the subscribing repositories.
+
+1. Test locally.
```

Comment:
Please declare this step as optional. In 90% of the cases you don't need to test runtime-assets changes locally before submitting a PR.